utils/python/CIME/XML/env_mach_pes.py

```diff
@@ -14,3 +14,29 @@ def __init__(self, case_root=os.getcwd(), infile="env_mach_pes.xml"):
         initialize an object interface to file env_mach_pes.xml in the case directory
         """
         EnvBase.__init__(self, case_root, infile)
+
+    def get_value(self, vid, attribute={}, resolved=True, subgroup=None):
```

Comment:
negative values in these fields correspond to a number of nodes rather than number of tasks.